src/main/java/games/strategy/engine/framework/headlessGameServer/AvailableGames.java

```diff
@@ -51,17 +49,11 @@ private static void populateFromDirectory(
       final Map<String, URI> availableGames,
       final Set<String> availableMapFolderOrZipNames) {
     final File games = new File(mapDir, "games");
-    if (!games.exists()) {
-      // no games in this map dir
-      return;
-    }
-    if (games.listFiles() != null) {
-      for (final File game : games.listFiles()) {
-        if (game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
-          final boolean added = addToAvailableGames(game.toURI(), availableGames);
-          if (added) {
-            availableMapFolderOrZipNames.add(mapDir.getName());
-          }
+    for (final File game : FileUtils.listFiles(games)) {
+      if (game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
+        final boolean added = addToAvailableGames(game.toURI(), availableGames);
+        if (added) {
+          availableMapFolderOrZipNames.add(mapDir.getName());
```

Comment:
Unrelated as well, but this could be converted to a nice functional-programming style stream, and added to the given set at once.